src/main/resources/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -28,6 +28,12 @@
     "name": "Google Cloud Resource Manager",
     "id": "googlecloudresourcemanager",
     "serviceName": "cloudresourcemanager.googleapis.com",
+    "serviceRoles": [
+      "roles/resourcemanager.organizationAdmin",
```

Comment:
why always the prefix "roles/"?

Reply:
because thats how they are identified by GCP (and how we identify them in IAM API calls).

It seems like this path (/) pattern is common to all resources in GCP (not just roles).